chapter13/sxg/ex_13_1.pl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+print "Enter directory structure: ";
+chomp( my $dir = <STDIN> );
+if ( $dir eq "" ) {
+	chdir or die "Can't chdir to home:  $!";
+}
+else {
+	chdir $dir or die "Can't chdir to '$dir': $!";
+}
+
+&print_file_from_current_dir();

```

Comment:
I thought it may make it more readable. 
- logical separation of code
- I have problem in looking at the a subroutine with 1000s of lines :)

May be this is not the way perl programmers want things, but I like to have small methods/functions all the time.